packages/jazz-tools/src/tests/coMap.test.ts

```diff
@@ -469,6 +474,66 @@ describe("CoMap resolution", async () => {
     expect(loadedPerson.dog?.name).toEqual("Rex");
   });
 
+  test("loading a locally available map with skipRetry set to true", async () => {
+    const Dog = co.map({
+      name: z.string(),
+      breed: z.string(),
+    });
+
+    const Person = co.map({
+      name: z.string(),
+      age: z.number(),
+      dog: Dog,
+    });
+
+    // Disconnect the current account
+    const currentAccount = Account.getMe();
+    currentAccount._raw.core.node.syncManager.getPeers().forEach((peer) => {
+      peer.gracefulShutdown();
+    });
```

Comment:
Considering that the load is called on the same node where the value is created we don't need to disconnect anything